src/Cache.php

```diff
@@ -56,9 +56,9 @@ final class Cache implements CacheInterface
     private $defaultTtl;
 
     /**
-     * @param \Psr\SimpleCache\CacheInterface cache handler.
+     * @param \Psr\SimpleCache\CacheInterface $handler cache handler.
```

Comment:
This could be removed completely - it repeats what obvious from method signature.